### DISCUSSION TITLE
Mac Viewer Apps

### DISCUSSION THREAD

**Discussion by donscreen** [GerryDazoo/Slinger#124] [General]
I am using a Macbook Air.   Gerry's server runs perfectly.   VLC on the Mac hangs sometimes...connects to a black screen or frozen picture on occasion.   I have found that MK Player and MPV work 100% of the time on the Mac.   My server is running on a Windows 10 machine in my home office.

**Comment by colfra** [comment]
I found that disabling hardware encoding in VLC's settings fixed the problem (on Macbook Pro and iOS/iPadOS devices too)

**Comment by shoobydoodoo** [comment]
Hi, I was searching for an usable Mac viewer because VLC simply did not work for me. Came upon your post and tried MKPlayer. It works beautifully without issues. Thank you so much for sharing your finding here and helping out fellow slingers. Merry Christmas!
